eng/common/pipelines/templates/steps/install-pipeline-generation.yml

```diff
@@ -3,7 +3,8 @@ parameters:
 
 steps:
   - script: >
-      mkdir $(Pipeline.Workspace)/pipeline-generator
+      mkdir pipeline-generator
+    workingDirectory: $(Pipeline.Workspace)
```

Comment:
Will this work if someone overrides the ToolPath?

Reply:
Yes ... they are actually unrelated. The purpose of creating the folder is to avoid ```dotnet``` thinking any MSBuild files in the directory or sub-directories are related to the tool install.

Reply:
It just so happens that in my usage I made then the same directory ;)